static/json-schema-viewer/index.html

```diff
@@ -156,83 +156,20 @@ <h1>Error!</h1>
-                    <li>
-                        <a href="http://jquerymobile.com/">jQuery Mobile</a>
-                    </li>
-                    <li>
-                        <a href="http://geraintluff.github.io/tv4/">tv4</a>
-                    </li>
-                    <li>
-                        <a href="http://bgrins.github.io/filereader.js/">FileReader.js</a>
-                    </li>
-                    <li>
-                        <a href="https://highlightjs.org/">highlight.js</a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/alexeykuzmin/jsonpointer.js">jsonpointer.js</a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/mhulse/grunt-html-boiler">Grunt HTML Boiler</a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/medialize/URI.js">URI.js</a>
-                    </li>
-                </ul>
-                <hr />
-                <h3> Features: </h3>
-                <ul>
-                    <li>
-                        <h4>Viewer</h4>
-                        <p>
-                            Interactive documentation tool for <a href="http://json-schema.org/">JSON schemas</a>.
-                            Utilizes the D3.js library to build a collapsible tree from the supplied schema(s).
-                        </p>
-                    </li>
-                    <li>
-                        <h4>JSON Validator</h4>
-                        <p>
-                            A tool to validate a json object using the supplied schema.
-                            Utilizes the tv4 library.
-                        </p>
-                    </li>
-                </ul>
-                <p>There's also a <a rel="external" href="./basic.html">basic example</a> without the jQuery Mobile interface elements.</p>
-                <hr />
-                <div id="welcome-footer">
-                    <a href="https://github.com/jlblcc/json-schema-viewer" class="ui-btn ui-btn-inline ui-icon-github ui-btn-icon-left">This project's on GitHub!</a>
-                </div>
-            </div>
-        </div>
-
         <script type='text/javascript' src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
         <script type="text/javascript" src="lib/preinit.js"></script>
         <script type='text/javascript' src="https://cdnjs.cloudflare.com/ajax/libs/jquery-mobile/1.4.5/jquery.mobile.min.js"></script>
         <script type='text/javascript' src="lib/json-schema-viewer.min.js"></script>
         <script>
             // reloads page, such that schema is displayed
+            /*
             window.onload = function() {
                 if(!window.location.hash) {
                     window.location = window.location + '#loaded';
                     window.location.reload();
                 }
-            }
+            }*/
```

Comment:
@zzeekk: are you sure this can be commented out? While I was testing it, the graph was not displayed on the screen.